### DISCUSSION TITLE
EDA tools

### DISCUSSION THREAD

**Discussion by PeterZhang17** [PeterZhang17/HelloEDA#3] [Q&A] [answered]
What are this EDA list for open-source hardware development?

**Comment by neilwang0913** [comment, marked as answer]
@SemiWiki
[https://semiwiki.com/wikis/industry-wikis/eda-open-source-tools-wiki/]

Comparison of EDA software @wiki
[https://en.wikipedia.org/wiki/Comparison_of_EDA_software]
